src/linux/linux-host-session.vala

```diff
@@ -419,7 +408,47 @@ namespace Frida {
 			return result;
 		}
 
-		public async uint spawn (string package_name, string? activity_name) throws Error {
+		public async uint spawn (string program, HostSpawnOptions options) throws Error {
+
+			string package_name = program;
+
+			var aux_options = options.load_aux ();
+
+			string  method = "activity";

```

Comment:
Let's avoid vertically aligning code, as it makes the style fragile wrt. future refactoring.

Reply:
ok